### DISCUSSION TITLE
Telemetry Mirror - Support for INAV 5.0

### DISCUSSION THREAD

**Discussion by FRPProject** [OpenI6X/opentx#267] [Q&A]
I have been able to properly run the telemetry mirror option using the PA9 pin, HC05, and Telemetry Viewer app. It works very nicely with Betalight 4.3, however had not much success with INAV 5.0. It seems OpenTX i6X (1.6.1) does not communicate with the flight controller to discover the sensors. I have enabled telemetry on the INAV configurator. Any suggestions would be greatly appreciate it. Thank you very much.

**Comment by ajjjjjjjj** [comment]
Do you see sensors on telemetry tab? I was testing it on inav 4.0 and it was fine. Make sure that your wiring is correct.

**Reply by FRPProject** [reply]
Thanks Janek; I have been flying this setup with no issues. On the telemetry tab I only see the generic sensors and not GPS. I think this has to do something with INAV 5.0.

**Reply by FRPProject** [reply]
@ajjjjjjjj I am getting back to you about this issue that I have had for awhile now. Have you been able to add INAV telemetry to openTX-i6x yourself?

**Reply by ajjjjjjjj** [reply]
I haven't upgraded to Inav 5.x yet, but I'm pretty sure it is configuration problem because nothing important regarding telemetry has changed. Have you seen this reply and thread? https://github.com/iNavFlight/inav/issues/6931#issuecomment-1030275248

I'll post again when i'll upgrade my inav setup.

**Reply by FRPProject** [reply]
Thanks for your reply. I am gonna look into the iNav thread a bit further. I'll get back to you if I resolve the issue.

**Reply by ajjjjjjjj** [reply]
I have updated to inav 5.1 and see all the sensors available, even without GPS connected i have 19 sensors including gyro data.
You may want to follow inav manual and check again all receiver/serial ports settings.

**Reply by FRPProject** [reply]
Fantastic, I was trying to mess around with UARTs. Thanks, I'll let you know if the problem is fixed or not.
